requirements.txt

```diff
@@ -0,0 +1,22 @@
+########################
```

Comment:
None of this are actual requirements. All of this are for testing/QA please rename this file as `requirements.dev.txt`